fiftyone/utils/transformers.py

```
@@ -1073,6 +1079,172 @@
+                x, y, w, h = detection.bounding_box
+                box_coco = [x * img_width, y * img_height, w * img_width, h * img_height]
+                
+                # Process with VitPose
+                inputs = self._transforms(img, boxes=[[box_coco]], return_tensors="pt")
+                inputs = {k: v.to(self._device) for k, v in inputs.items()}
+                
+                with torch.no_grad():
+                    outputs = self._model(**inputs)
+                
+                # Post-process
+                pose_results = self._transforms.post_process_pose_estimation(
```

Pylint (3.3.7)

[error] Instance of '_HFTransformsHandler' has no 'post_process_pose_estimation' member

(E1101)